src/drivers/boards/omnibus-f4sd/spi.c

```diff
@@ -0,0 +1,194 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2012 PX4 Development Team. All rights reserved.
```

Comment:
Update

Reply:
sure, just the date or should i add my name under the copyright?

Reply:
That's up to you. Personally I use PX4 Development Team, but you're welcome to use your name if you'd prefer. Either way with the current date.

Reply:
on it. thx!